docs/src/contributing.md

```diff
@@ -2,33 +2,63 @@
 
 This guide contains tips and tricks for working on Shotover Proxy itself.
 
-## Building Shotover
+## Configuring the Environment
 
 Shotover is written in Rust, so make sure you have a rust toolchain installed. See [the rustup site](https://rustup.rs/) for a quick way to setup your Rust development environment.
 
 Once you've installed Rust via Rustup (you should just be fine with the latest stable). You will need to install a few other tools needed to compile some of Shotover's dependencies.
 
 Shotover requires the following in order to build:
 
-* cmake
 * gcc
 * g++
 * libssl-dev
 * pkg-config (Linux)
+* Cassandra CPP Driver
 
-On ubuntu you can install them via `sudo apt-get install cmake gcc g++ libssl-dev pkg-config`
+On Ubuntu you can install all except the Cassandra driver via `sudo apt-get install cmake gcc g++ libssl-dev pkg-config`
 
+### Installing Cassandra CPP Driver
```

Comment:
Shouldnt this also go under `Installing Optional Tools and Libraries` as its only needed when running `cargo test`